pkg/apis/serving/v1alpha1/kfservice_status.go

```diff
@@ -14,41 +14,82 @@ limitations under the License.
 package v1alpha1
 
 import (
-	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+	"github.com/knative/pkg/apis"
 	knservingv1alpha1 "github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"k8s.io/api/core/v1"
 )
 
 // ConditionType represents a Service condition value
 const (
-	// ServiceConditionRoutesReady is set when the service's underlying
+	// KFServiceConditionRoutesReady is set when the service's underlying
 	// routes have reported readiness.
-	ServiceConditionRoutesReady duckv1alpha1.ConditionType = "RoutesReady"
-	// ServiceConditionDefaultConfigurationsReady is set when the service's underlying
+	KFServiceConditionRoutesReady apis.ConditionType = "RoutesReady"
+	// KFServiceConditionDefaultConfigurationsReady is set when the service's underlying
 	// default configuration have reported readiness.
-	ServiceConditionDefaultConfigurationsReady duckv1alpha1.ConditionType = "DefaultConfigurationReady"
+	KFServiceConditionDefaultConfigurationsReady apis.ConditionType = "DefaultConfigurationReady"
```

Comment:
I think this should just be "DefaultReady" for the same reason about exposing knative configurations through our api.

How do you feel about two for now:
"DefaultReady"
"DefaultPredictorReady"

Each of the components will need to ready up and they can rollup.